common/views/components/BaseHeader/ExhibitionHeaderReadme.md

```diff
@@ -0,0 +1,9 @@
+## Exhibition header
+
+This component comprises a `picture` element and a `HeaderText` component.
+
+The picture will render a square crop on viewports narrower than 600px, and the 'thin' (32:15) crop thereafter.
```

Comment:
Readme content like this is asking to go out of date really quick - shall we stick to the purpose and not the execution in docs to avoid that?

Reply:
Sounds sensible